de.dlr.sc.virsat.model.extension.cef.interfaces/src/de/dlr/sc/virsat/model/extension/cef/interfaces/model/AInterfaceEnd.java

```diff
@@ -63,11 +63,12 @@ public AInterfaceEnd(CategoryAssignment categoryAssignment) {
 	 */
 	public long getGlobalUnitQuantity() {
 		long globalUnitQuantity = 1;
-		try {
-			globalUnitQuantity *= getQuantity();
-		} catch (NumberFormatException e) {
-			globalUnitQuantity *= 1;
-		}
+
+		// Workaround due to API Inconsistency with Beans see https://github.com/virtualsatellite/VirtualSatellite4-CEF/issues/99

```

Comment:
Let's maybe do it something like
`globalUnitQuantity *= isSetQuantity() ? getQuantity() : 1;`